src/sentry/deletions/manager.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import, print_function
+
+__all__ = ['DeletionTaskManager']
+
+
+class DeletionTaskManager(object):
+    def __init__(self, default_task=None):
+        self.tasks = {}
+        self.default_task = default_task
+
+    def get(self, task=None, **kwargs):
```

Comment:
this feels a little odd, and was needed since we cascade the manager into tasks (so its not using a singleton).